api/src/entities/Multiflash.py

```diff
@@ -1,10 +1,11 @@
 from typing import Dict, NamedTuple
 
-import libhg
 import numpy as np
 import numpy.typing as npt
 from pydantic import BaseModel, Field, validator
 
+import libhg
```

Comment:
again, why?

Reply:
Issues with isort. Looks to be fixed now.